task05/task05/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task05
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            var x = Calculate(1, 2, 3) + Calculate(5, 3, 8) + Calculate(1, 5, 6);
+            Console.WriteLine(Math.Round(x, 3));
+        }
+
+        static double Calculate(double a, double b, double c) =>
+            Math.Sqrt((a + Math.Pow(Math.Tan(b),2)) / c);
```

Comment:
Можно было обойтись двумя параметрами
```suggestion
        static void Main(string[] args)
        {
            var x = Calculate(1, 2) + Calculate(5, 3) + Calculate(1, 5);
            Console.WriteLine(Math.Round(x, 3));
        }

        static double Calculate(double a, double b) =>
            Math.Sqrt((a + Math.Pow(Math.Tan(b),2)) / (a + b));
```